src/routes/admin/parts/+page.server.js

```diff
@@ -1,8 +1,8 @@
 import {getUserRoles} from "$lib/server/account.js";
 import {redirect} from "@sveltejs/kit";
+import {getAllParts} from "../../../lib/server/parts.js";
```

Comment:
On peut raccourcir l'import ici.
```suggestion
import {getAllParts} from '$lib/server/parts.js';
```